org.kernel.TCMUService1.service

```diff
@@ -3,6 +3,6 @@
 
 [D-BUS Service]
 Name=org.kernel.TCMUService1
-Exec=/usr/bin/tcmu-runner
+Exec=/usr/local/bin/tcmu-runner
```

Comment:
do you mean changes in  tcmu-runner.service file?
Because make install do install tcmu-runner binary on /usr/local/bin/ path

Is that what you are asking?